packages/cli/src/commands/hydrogen/build.ts

```diff
@@ -41,6 +45,10 @@ export default class Build extends Command {
       env: 'SHOPIFY_HYDROGEN_FLAG_SOURCEMAP',
       default: false,
     }),
+    ['hide-bundle-stats']: Flags.boolean({
+      description: 'Hide the bundle size analysis.',
+      default: false,
+    }),
```

Comment:
As we talked, this could just be `bundle-stats` with `allowNo: true`. We do this for `installDeps` as well.
There are also another similar pattern that we have: `disableRouteWarning` and `disableVirtualRoutes`.

This case might be more similar to the latter but maybe we should just update all of them to use `allowNo` instead? 🤔